shell/ci/release/confluence-publish.sh

```diff
@@ -75,4 +75,5 @@ while read -r file; do
   else
     info_sub "no space directive found, skipping ${file}"
   fi
-done < <(git diff --name-only HEAD HEAD~1 -- '*.md')
+  readarray -t releaseCommits < <(git log --pretty=format:'%H' -n 2 --grep='^chore' --invert-grep)
```

Comment:
Instead of doing it based on a list of commits, we probably want to change the behaviour to be just on release. Otherwise we'd have to maintain two different lists of what releases/what doesn't.

I'm also concerned about this because we likely have the chance we'd move away from conventional commits in the future in favor of more explicit control of what version should be changed as part of the PR. e.g. specifying "bump: minor" since generally the conventional commit rules have been confusing to people.

Reply:
Was thinking of a separate issue, this is completely fine with the current grpcClients stuff. So, this is fine.